crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -139,6 +144,11 @@ def adminrepocheck
           end
 
           unless zypper_stream["prompt"].nil?
+            upgrade_status.end_step(
+              false,
+              data: zypper_stream["prompt"]["text"],
+              help: "Make sure you solve the required action and try again."
```

Comment:
I think we're solving problems, not actions. But I'm not sure what the error message actually says...